x-pack/plugins/apm/public/components/app/service_groups/service_group_save/select_services.tsx

```diff
@@ -78,6 +78,19 @@ export function SelectServices({
     }
   }, [isEdit, serviceGroup.kuery]);
 
+  useEffect(() => {
+    if (!stagedKuery) {
+      return;
+    }
+    const { isValid, isParsingError, message } =
+      validateServiceGroupKuery(stagedKuery);
+    if (isValid || isParsingError) {
+      setKueryValidationMessage(undefined);
+    } else {
+      setKueryValidationMessage(message);
+    }
```

Comment:
We don't care if there were errors on not - we want to update the message unconditionally:

```suggestion
    setKueryValidationMessage(message);
```